rm-datasource/src/main/java/io/seata/rm/datasource/DataSourceProxy.java

```diff
@@ -143,6 +153,32 @@ private boolean isPolardbXProduct() {
         return false;
     }
 
+    /**
+     * check existence of undolog table
+     *
+     * if the table not exist fast fail, or else keep silence
+     *
+     * @param conn db connection
+     */
+    private void checkUndoLogTableExist(Connection conn) {
+        if (DBType.optionalof(dbType).isPresent()) {
+            UndoLogManager undoLogManager;
+            try {
+                undoLogManager = UndoLogManagerFactory.getUndoLogManager(dbType);
+            } catch (EnhancedServiceNotFoundException e) {
+                LOGGER.error("can't find undoLogManager service provider for dbtype: {}", dbType);
```

Comment:
我建议这里直接抛异常出去，说明用户再使用一个我们不支持的数据库，要提示这个数据库不支持
I suggest throwing an exception directly here, indicating that if the user uses a database that we do not support, we need to prompt that the database is not supported.

Reply:
没有抛出去，是因为很多单测使用了数据库代理，会导致单测失败。
It is not thrown out because many unit tests use database proxy, which can be lead it fail.

Reply:
> 没有抛出去，是因为很多单测使用了数据库代理，会导致单测失败。 It is not thrown out because many unit tests use database proxy, which can be lead it fail.

有没有什么办法在单测的时候不进行校验undolog table？或者增加一个开关，这个开关允许用户配置check，如果用户关闭，就不进行check，然后我们默认开启，在单测的时候关闭check
Is there any way to not check undolog table during single test or add a switch that allows user to configure check and if user turns it off, no check is performed, then we turn it on by default and turn off check during single test

Reply:
update mockito version to 4.5.1 same as seata server's mockito dependency to support static mock. when unit test. mock undolog table check.